### PR TITLE
[Cards] add missing theming extension doc.

### PR DESCRIPTION
**Pull request by fluxminds**
Updated unused theming.md to reflect theming extension documentation.

### CODE REVIEW

File: components/Cards/README.md [hunk 1]
```diff
@@ -28,10 +28,25 @@ purposes.
...
   <li class="icon-list-item icon-list-item--link">Enumeration: <a href="https://material.io/components/ios/catalog/cards/api-docs/Enums/MDCCardCellHorizontalImageAlignment.html">MDCCardCellHorizontalImageAlignment</a></li>
   <li class="icon-list-item icon-list-item--link">Enumeration: <a href="https://material.io/components/ios/catalog/cards/api-docs/Enums/MDCCardCellState.html">MDCCardCellState</a></li>
   <li class="icon-list-item icon-list-item--link">Enumeration: <a href="https://material.io/components/ios/catalog/cards/api-docs/Enums/MDCCardCellVerticalImageAlignment.html">MDCCardCellVerticalImageAlignment</a></li>
+  <li class="icon-list-item icon-list-item--link">Enumeration: <a href="https://material.io/components/ios/catalog/cards/api-docs/Enums/MDCInkStyle.html">MDCInkStyle</a></li>
+  <li class="icon-list-item icon-list-item--link">Enumeration: <a href="https://material.io/components/ios/catalog/cards/api-docs/Enums/MDCRippleState.html">MDCRippleState</a></li>
+  <li class="icon-list-item icon-list-item--link">Enumeration: <a href="https://material.io/components/ios/catalog/cards/api-docs/Enums/MDCRippleStyle.html">MDCRippleStyle</a></li>
```
Comment (jverkoey) [inline components/Cards/README.md]: Same bug here: https://github.com/material-components/material-components-ios/issues/7195

Please revert this section's changes manually for now.

Comment (fluxminds) [reply components/Cards/README.md]: Reverted in e26b9e4a532da6e8bcc5adb59842f08ff63bca94

File: components/Cards/docs/theming-extensions.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+### Theming Extensions
```
Comment (jverkoey) [inline components/Cards/docs/theming-extensions.md]: Please rename to “Theming” and theming.md

File: components/Cards/docs/theming-extensions.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ ...
+ // Create or use your app's Container Scheme
+MDCContainerScheme *containerScheme = [[MDCContainerScheme alloc] init];
+ // Theme the card with either default theme
+[self.card applyThemeWithScheme:containerScheme];
+ // Or outlined theme
+[self.card applyOutlinedThemeWithScheme:containerScheme];
+```
+
+<!--</div>-->
```
Comment (jverkoey) [inline components/Cards/docs/theming-extensions.md]: Please add this file to docs/README.md and re-run the generator.

File: components/Cards/docs/theming.md [hunk 4]
```diff
@@ -1,38 +1,37 @@
 ### Theming
```
Comment (romoore) [inline components/Cards/docs/theming.md:1]: Looks like this file was never included in README.md. Could you please add it and make sure it appears in the generated output?

File: components/Cards/docs/theming-extensions.md (hunk 3 above)
Comment (fluxminds) [reply components/Cards/docs/theming-extensions.md]: Addressed in a20a897 and 0503c1c. Description updated.

File: components/Cards/docs/theming-extensions.md (hunk 2 above)
Comment (fluxminds) [reply components/Cards/docs/theming-extensions.md]: Addressed in a20a897 and 0503c1c. Description updated.

File: components/Cards/docs/theming.md (hunk 4 above)
Comment (fluxminds) [reply components/Cards/docs/theming.md:1]: Addressed in 0503c1c

Comment (romoore) [reply components/Cards/docs/theming.md:1]: Ah, nevermind. GitHub sent me to a single commit, not the PR.

**Review by jverkoey — APPROVED** [review]

**Pull request merged**